Kernel/Library/ThreadSafeRefCounted.h

```diff
@@ -98,9 +100,9 @@ class RefCounted : public RefCountedBase {
             call_will_be_destroyed_if_present(static_cast<const T*>(this));
             delete static_cast<const T*>(this);
             return true;
-        } else if (new_ref_count == 1) {
-            call_one_ref_left_if_present(static_cast<const T*>(this));
         }
+        if (new_ref_count == 1)
+            call_one_ref_left_if_present(static_cast<const T*>(this));
```

Comment:
Nitpick: wrong commit

Reply:
? The comment is on line 38, this is in line 101. I think the comment is correct.

Reply:
This particular diff around the lines 101-105 belongs into the "Avoid else after return" commit, instead of the "Suppress clang-tidy warnings from the cert-* category" commit. I'm not talking about line 38.

Reply:
Whoops good catch. Forgot this was in this file when I was splitting commits

Reply:
Apologies, my mind read "commit" as "comment"...